autoscale-api/src/main/java/com/sequenceiq/periscope/api/model/DistroXAutoscaleScalingActivityResponse.java

```diff
@@ -0,0 +1,72 @@
+package com.sequenceiq.periscope.api.model;
+
+import java.util.Date;
+
+import com.sequenceiq.periscope.doc.ApiDescription;
+
+import io.swagger.annotations.ApiModel;
+import io.swagger.annotations.ApiModelProperty;
+
+@ApiModel
+public class DistroXAutoscaleScalingActivityResponse implements Json {
+
+    @ApiModelProperty(ApiDescription.ClusterJsonsProperties.ACTIVITY_CRN)
+    private String activityCrn;
+
+    @ApiModelProperty(ApiDescription.ClusterJsonsProperties.START_TIME)
+    private Date startTime;
+
+    @ApiModelProperty(ApiDescription.ClusterJsonsProperties.END_TIME)
+    private Date endTime;
+
+    @ApiModelProperty(ApiDescription.ClusterJsonsProperties.ACTIVITY_STATUS)
+    private ApiActivityStatus apiActivityStatus;
+
+    @ApiModelProperty(ApiDescription.ClusterJsonsProperties.ACTIVITY_REASON)
+    private String scalingActivityReason;
+
+    public DistroXAutoscaleScalingActivityResponse() {
+    }
+
+    public String getActivityCrn() {
+        return activityCrn;
+    }
+
+    public void setActivityCrn(String activityCrn) {
+        this.activityCrn = activityCrn;
+    }
+
+    public void setStartTime(Date startTime) {
+        this.startTime = startTime;
+    }
+
+    public Date getStartTime() {
+        return startTime;
+    }
+
+    public void setEndTime(Date endTime) {
+        this.endTime = endTime;
+    }
+
+    public Date getEndTime() {
+        return endTime;
+    }
+
+    public ApiActivityStatus getActivityStatus() {
+        return apiActivityStatus;
+    }
+
+    public void setActivityStatus(ActivityStatus activityStatus) {
+        ActivityStatusToApiActivityStatusConverter activityStatusToApiActivityStatusConverter = new ActivityStatusToApiActivityStatusConverter();
+        this.apiActivityStatus = activityStatusToApiActivityStatusConverter.convert(activityStatus);
+    }
```

Comment:
This setter doesn't look right. It should just set the `ApiActivityStatus` when provided with an argument of the same type

Reply:
I have merged both the converter functions into one.